app/controllers/resources_controller.rb

```diff
@@ -1,16 +1,37 @@
 class ResourcesController < ApplicationController
 
   def resource_params
-    params.require(:resources).permit(:title, :url, :contact_email, :location, :types, :audiences, :presence)
+    params.permit(:title, :url, :contact_email, :location, :types, :audiences, :population_focuses, :campuses,
```

Comment:
shouldn't we require the required parameters?

Reply:
the validations are in the resource.rb file!

 I believe this method in the controller is filtering the parameters in the API request itself.